tests/scriptcomp/corpus/variables.nss

```diff
@@ -0,0 +1,29 @@
+void main()
+{
+    int a = 1;
+    int b = 2;
+    int c = 3;
+    int d = 4;
+    string s = "A";
+    float f = 1.0f;
+
+    if (1)
+    {
+        int a = 10;
+        int b = 20;
+        int c = 30;
+        int d = 40;
+        string s = "B";
+        float f = 10.0f;
+
+        Assert(a = 10);
```

Comment:
=, ==?

Reply:
Ouch. That makes all my testing invalid. Fortunately, it still passes when fixed :)